### ISSUE TITLE
Ganglionic AchR antibodies need separate term from neuromuscular junction type

### ISSUE THREAD

**Issue by MickeySegal** [obophenotype/human-phenotype-ontology#9632]
**Preferred term label:**
Anti-acetylcholine receptor (ganglionic) antibody positivity
 Anti-acetylcholine receptor (neuromuscular junction) antibody positivity
**Synonyms**
Anti-AChR
**Definition (free text, please give PubMed ID)**
The presence of autoantibodies (immunoglobulins) in the blood circulation that react against acetylcholine receptors of the ganglionic type
The presence of autoantibodies (immunoglobulins) in the blood circulation that react against acetylcholine receptors of the neuromuscular junction type
**Parent term (use hpo.jax.org/app)**
HP:0030208 could be the parent term, though its diseases are characteristic of the neuromuscular junction type and perhaps it should be a sibling term and a new parent term created for both ganglionic and neuromuscular junction types.

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
For the ganglionic type: 
https://pubmed.ncbi.nlm.nih.gov/14732619/ : Idiopathic autonomic neuropathy
https://pubmed.ncbi.nlm.nih.gov/18474849/ Thymoma

Several labs offer tests for the ganglionic type:
https://www.athenadiagnostics.com/view-full-catalog/g/ganglionic-achr-antibody-test
https://ltd.aruplab.com/Tests/Pub/3003020
https://testdirectory.questdiagnostics.com/test/test-detail/93881/acetylcholine-receptor-ganglionic-alpha-3-antibody?cc=MASTER

UMLS has a term for the ganglionic "laboratory procedure", but it is ambiguous as to whether it also refers to its positivity: C4737384 "acetylcholine receptor ganglionic neuronal antibody in serum"

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
@MickeySegal 
I have rearranged as follows
1. New term
HP:0025762, Anti-ganglionic acetylcholine receptor antibody positivity
2. I have renamed the existing term HP:0030208  Anti-neuromuscular junction acetylcholine receptor antibody positivity (because the diseases are annotated to this)
3. I have created a new parent term HP:6001064 Anti-acetylcholine receptor antibody positivity

**Comment by MickeySegal** [comment]
Does this change flow through to UMLS or do we need to give them a heads-up?
